LayoutTests/http/tests/navigation/keyboard-events-during-provisional-subframe-navigation-expected.txt

```diff
@@ -8,6 +8,8 @@ CONSOLE MESSAGE: Pressing "a".
 CONSOLE MESSAGE: Setting marked text to "b".
 CONSOLE MESSAGE: Inserting text "c".
 CONSOLE MESSAGE: Pasting text "d".
+CONSOLE MESSAGE: pasteevent dispatched (isTrusted: true).
```

Comment:
Perhaps this could also verify that the data is inaccessible as well? (i.e., verify that `DataTransfer.getData("text/plain")`, `DataTransfer.types` and `DataTransfer.items` all return empty strings/arrays).

Reply:
Yeah, I can add this. Maybe line 13 below already tests part of this? Without the change in WebPasteBoardProxy line 13 turns into `CONSOLE MESSAGE: Input element value after text input events: "d".` (d is pasted)

Reply:
Ah, I see! It should be fine as-is, then.